### PR TITLE
Add packages

### PR DESCRIPTION
**Pull request by johnpryan**

### CODE REVIEW

File: pkgs/samples/lib/gemini.dart [hunk 1]
```diff
@@ -0,0 +1,336 @@
+// Copyright 2024 the Dart project authors. All rights reserved.
```
Comment (devoncarew) [inline pkgs/samples/lib/gemini.dart:1]: Should this be a built-in sample or one that we provide via a gist?

**Review by devoncarew — APPROVED** [review]
lgtm, though we may want the sample to be available via a gist instead?

File: pkgs/samples/lib/gemini.dart (hunk 1 above)
Comment (johnpryan) [reply pkgs/samples/lib/gemini.dart]: I'll take it out for now. Here's the gist: https://gist.github.com/flutterdevrelgists/341bc46b2ed1d2055d357ab987ed5fc2

**Comment by parlough** [comment]
Nice! Glad it's working out :D

**Comment by parlough** [comment]
@johnpryan Did you get the link portion working? I remember we had trouble in the past and might need to further configure the iframe sandbox options?

**Pull request merged**